packages/web/src/common/store/pages/token-dashboard/getAccountMetadataCID.ts

```diff
@@ -7,14 +8,16 @@ const { getUserId } = accountSelectors
 
 export function* getAccountMetadataCID() {
   yield* waitForRead()
-  const apiClient = yield* getContext('apiClient')
+  const sdk = yield* getSDK()
   const accountUserId = yield* select(getUserId)
   if (!accountUserId) return null
 
-  const users = yield* call([apiClient, apiClient.getUser], {
-    userId: accountUserId,
-    currentUserId: accountUserId
+  const { data = [] } = yield* call([sdk.full.users, sdk.full.users.getUser], {
```

Comment:
this is so strange - `getUser` returns an array?

Reply:
Yep the API does! I believe this might be for batching?